### PR TITLE
Fixed some warnings in Time series weather forecasting

### PR DESCRIPTION
**Pull request by mohantym**
Fixed some Typos, Markdown warnings, Doc string in Time_series_weather_Forecasting.

### CODE REVIEW

File: examples/timeseries/md/timeseries_weather_forecasting.md [hunk 1]
```diff
@@ -38,24 +38,23 @@ in Jena, Germany
...
-8    |VPact (mbar)   |3.11               |Vapor pressure
-9    |VPdef (mbar)   |0.22               |Vapor pressure deficit
-10   |sh (g/kg)      |1.94               |Specific humidity
-11   |H2OC (mmol/mol)|3.12               |Water vapor concentration
-12   |rho (g/m ** 3) |1307.75            |Airtight
-13   |wv (m/s)       |1.03               |Wind speed
-14   |max. wv (m/s)  |1.75               |Maximum wind speed
-15   |wd (deg)       |152.3              |Wind direction in degrees
-
+| Index | Features        | Format              | Description                                                                                                                                                                                                        |
```
Comment (fchollet) [inline examples/timeseries/md/timeseries_weather_forecasting.md:41]: The table was dramatically more readable before! Why change it?

**Review by fchollet — COMMENTED** [review]
Thanks for the PR -- unfortunately it doesn't look very high value right now.

File: examples/timeseries/md/timeseries_weather_forecasting.md (hunk 1 above)
Comment (mohantym) [reply examples/timeseries/md/timeseries_weather_forecasting.md:41]: Sorry! Just followed suggestion from Pycharm Editor on readability. Thanks for the feedback.

**Comment by fchollet** [comment]
Let's close, sounds good.

**Pull request closed without merging**